Dockerfile

```diff
@@ -65,6 +65,7 @@ RUN echo "import sys\nsys.path.append('/source/python')" > /etc/python2.7/sitecu
 
 ADD tools/docker/supervisord.conf /etc/supervisor/conf.d/nav.conf
 
+COPY requirements/ /requirements
```

Comment:
Most, if not all, of the `ADD`s in our Dockerfiles should probably be changed to `COPY`.